apps/nextjs-website/src/components/atoms/ChatMessage/ChatMessage.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Stack, Typography, useTheme } from '@mui/material';
+import PersonOutlineIcon from '@mui/icons-material/PersonOutline';
+import AdbOutlinedIcon from '@mui/icons-material/AdbOutlined';
+
+type ChatMessageProps = {
+  message: string;
+  sender?: string;
+  timestamp: string;
+};
+
+const ChatMessage = ({ message, sender, timestamp }: ChatMessageProps) => {
+  const { palette } = useTheme();
+  const bgColor = sender ? palette.background.paper : `${palette.info.light}80`;
+  const titleColor = palette.text.primary;
```

Comment:
this value is used only one time i think is better to asing value directly in tsx below